webapp/channels/src/packages/mattermost-redux/src/reducers/entities/integrations.ts

```diff
@@ -296,6 +296,15 @@ function appsBotIDs(state: string[] = [], action: MMReduxAction) {
     }
 }
 
+function dialogArguments(state = '', action: MMReduxAction) {
```

Comment:
The type of the default value here is incorrect since it doesn't match the type added below